src/api/spec/spec_helper.rb

```diff
@@ -20,6 +20,9 @@
     # ...rather than:
     #     # => "be bigger than 2"
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+
+    # to disable deprecated should syntax
+    expectations.syntax = :expect
```

Comment:
@ChrisBr I added the same line in my spec_helper.rb, so no need to change nothing :smile: